pkgs/by-name/gl/glas/package.nix

```
@@ -0,0 +1,35 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "glas";
+  version = "0.2.1";
+
+  src = fetchFromGitHub {
+    owner = "maurobalbi";
+    repo = "glas";
+    rev = "v${version}";
+    sha256 = "sha256-9RYCWGZ2BHL9SnX6cFXdTkzoWoP5AVkPCexQ1IWjD6s=";
+  };
+
+  cargoHash = "sha256-8PuZKTVqIM2bcQIID1bodYnCXDHz8UCXo/Y1odglL/U=";
+
+  doInstallCheck = true;
+  postInstallCheck = ''
+    $out/bin/glas --help > /dev/null
+  '';
+
+  meta = with lib; {
+    description = "A language server for the Gleam programming language.";
+    homepage = "https://github.com/maurobalbi/glas";
+    changelog = "https://github.com/maurobalbi/glas/tag/v${version}";
+    license = with lib.licenses; [
```

```suggestion
    license = with licenses; [
```
`lib`attributes are already in scope

---

removed `with lib;` as I remember that being unadvised.